frontend/src/components/visual-editor/form/inputs/triggers/PatternsInput.tsx

```
@@ -40,42 +73,19 @@ const StyledNoPatternsDiv = styled("div")(
   }),
 );
 
-type PatternsInputProps = {
-  value: Pattern[];
-  onChange: (patterns: Pattern[]) => void;
-  minInput: number;
-};
+type PatternsInputProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)